### PR TITLE
Add websocket messages for status querying ethereum transactions

### PR DESCRIPTION
**Pull request by yabirgb**
Adds websockets notifications to informt the user what is the current progress querying ethereum transactions.

At the moment a notification is displayed when the address that is being queried changes but more information is sent to the frontend via the websockets connection: 

 - `QUERYING_TRANSACTIONS`
 - `QUERYING_INTERNAL_TRANSACTIONS`
 - `QUERYING_ETHEREUM_TOKENS_TRANSACTIONS`

@lukicenturi We were discussing about adding this information to the ethereum transactions panel instead of showing notifications since they could blink and pile up if you have many accounts. What do you think of adding the information here when the websockets receives the change in step?

image

### CODE REVIEW

File: rotkehlchen/chain/ethereum/transactions.py [hunk 1]
```diff
@@ -115,7 +117,15 @@ def query(
             f_to_ts = filter_query.to_ts
             from_ts = Timestamp(0) if f_from_ts is None else f_from_ts
             to_ts = ts_now() if f_to_ts is None else f_to_ts
-            for address in accounts:
+            for i, address in enumerate(accounts):
```
Comment (LefterisJP) [inline rotkehlchen/chain/ethereum/transactions.py:120]: What is this ???

File: rotkehlchen/chain/ethereum/transactions.py [hunk 2]
```diff
@@ -148,6 +158,14 @@ def _get_transactions_for_range(
         dbethtx = DBEthTx(self.database)
         for query_start_ts, query_end_ts in ranges_to_query:
             log.debug(f'Querying Transactions -> {query_start_ts} - {query_end_ts}')
+            self.msg_aggregator.add_message(
```
Comment (LefterisJP) [inline rotkehlchen/chain/ethereum/transactions.py:161]: here and elsewhere remove the log.debug messages. The UI should be enough to check these from. And in tests or debugging even when there is no websockets/notifier it should be added to the legacy errors of the `msg_aggregator`

**Comment by lukicenturi** [comment]
> Adds websockets notifications to informt the user what is the current progress querying ethereum transactions.
> 
> At the moment a notification is displayed when the address that is being queried changes but more information is sent to the frontend via the websockets connection:
> 
> * `QUERYING_TRANSACTIONS`
> * `QUERYING_INTERNAL_TRANSACTIONS`
> * `QUERYING_ETHEREUM_TOKENS_TRANSACTIONS`
> 
> @lukicenturi We were discussing about adding this information to the ethereum transactions panel instead of showing notifications since they could blink and pile up if you have many accounts. What do you think of adding the information here when the websockets receives the change in step?
> 
> image

Alright thanks for the changes, I will work on it @yabirgb

File: rotkehlchen/chain/ethereum/transactions.py (hunk 1 above)
Comment (yabirgb) [reply rotkehlchen/chain/ethereum/transactions.py:120]: This is to notify the frontend the number of accounts that will be queried for this time range. So you know you are in the i aaccoutn of the `len(accounts)` What I did wrong is to use single letter variable

Comment (LefterisJP) [reply rotkehlchen/chain/ethereum/transactions.py:120]: I am sorry but I still do not understand what this is. What purpose does it serve? We don't care about this, do we? Where is this useful?

Comment (yabirgb) [reply rotkehlchen/chain/ethereum/transactions.py:120]: This is passed later to the frontend in the message https://github.com/rotki/rotki/pull/4330/files#diff-9e03fec7cdf8f66f69931ce6e1c42f6dd1c7e23218c45cacbcb4dbeb3424a3acR125

So the frontend knows that it has queried `i+1` addresses of `len(accounts)`(total)

File: rotkehlchen/chain/ethereum/transactions.py [hunk 3]
```diff
@@ -116,6 +118,14 @@ def query(
             from_ts = Timestamp(0) if f_from_ts is None else f_from_ts
             to_ts = ts_now() if f_to_ts is None else f_to_ts
             for address in accounts:
+                self.msg_aggregator.add_message(
```
Comment (LefterisJP) [inline rotkehlchen/chain/ethereum/transactions.py:121]: Again as discussed this is not needed.

**Comment by yabirgb** [comment]
Closing in favor of https://github.com/rotki/rotki/pull/4365

**Pull request closed without merging**